Userland/Libraries/LibCore/MimeData.cpp

```diff
@@ -54,56 +54,45 @@ void MimeData::set_text(DeprecatedString const& text)
     set_data("text/plain", text.to_byte_buffer());
 }
 
+static AK::Vector<StringView> s_plaintext_suffixes = {
```

Comment:
Small nit, both of these can be Array instead of Vector

Reply:
Fixed